include/deal.II/base/mpi.h

```diff
@@ -1193,70 +1193,36 @@ namespace Utilities
             const unsigned int    root_process = 0);
 
     /**
-     * This function sends an object @p object_to_send from the process @p root_process
-     * to all other processes.
+     * This function sends an object @p object_to_send from the process @p
+     * root_process to all other processes.
      *
      * This function is a generalization of the classic `MPI_Bcast` function
-     * that accepts arbitrary data types `T`, as long as Utilities::pack()
-     * (which in turn uses `boost::serialize`, see in Utilities::pack() for
-     * details) accepts `T` as an argument.
-     *
-     * @note Be aware that this function is typically a lot more
-     * expensive than an `MPI_Bcast` because the function will use
-     * boost::serialization to (de)serialize, and execute a second
-     * `MPI_Bcast` to transmit the size before sending the data
-     * itself. On the other hand, if you have a single element of
-     * a data type `T` that is natively supported by MPI, then the
-     * compiler will choose another broadcast() overload that is efficient.
-     * If you have an array of such elements, you should use the other
-     * broadcast() function in this namespace that takes a pointer and a count
-     * argument.
```

Comment:
I think this might be nice to retain.

Reply:
Good idea - I fixed a few references in the text so it makes sense in a unified function.